bin/yaml/tools.yaml

```diff
@@ -9,13 +9,28 @@ tools:
     check_exe: bin/cmake --help
     targets:
       - 3.26.1
-      - 3.26.4
-      - 3.27.9
+      - name: 3.26.4
+        symlink: cmake-v3.26
+      - name: 3.27.9
+        symlink: cmake-v3.27
       - 3.28.0-rc5
       - name: 3.28.0
         symlink: cmake  # The one distinguished cmake we use for "cmake" on the site
-      - 3.28.4
-      - 3.29.2
+      - name: 3.28.4
+        symlink: cmake-v3.28
+      - name: 3.29.2
+        symlink: cmake-v3.29
+  cmake-experimental:
```

Comment:
please indent this whole section once and name it `experimental` - there's no need to have `cmake` and `cmake-experimental` as two top-level packages: `cmake/experimental` is a fine way to separate versions.

I tested this install and it seems to work - so thank you.